async/delay.ts

```diff
@@ -1,9 +1,16 @@
 // Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
 /* Resolves after the given number of milliseconds. */
-export function delay(ms: number): Promise<void> {
-  return new Promise((res): number =>
-    setTimeout((): void => {
-      res();
-    }, ms)
-  );
+export function delay(ms: number, signal?: AbortSignal): Promise<void> {
```

Comment:
I think the 2nd argument should be an object of the shape of `options?: { signal?: AbortSignal }`. This design makes the future addition of other optional parameters hard. We have a style guideline for this rule https://deno.land/manual/contributing/style_guide#exported-functions-max-2-args-put-the-rest-into-an-options-object

Reply:
Good point, will do.